react-example/src/views/InApp.tsx

```diff
@@ -0,0 +1,149 @@
+import { FC, FormEvent, useState } from 'react';
+import styled from 'styled-components';
+import _Button from 'src/components/Button';
+import { EndpointWrapper, Form, Heading, Response } from './Components.styled';
+import { useUser } from 'src/context/Users';
+import { getInAppMessages } from '@iterable/web-sdk';
+
+const Button = styled(_Button)`
+  width: 100%;
+  margin-bottom: 1em;
+`;
+
+const GetMessagesRawButton = styled(Button)`
+  width: 45%;
+  max-height: 61px;
+
+  @media (max-width: 850px) {
+    width: 100%;
+  }
+`;
+
+const AutoDisplayContainer = styled.div`
+  width: 65%;
+  margin: 0 auto;
+
+  @media (max-width: 850px) {
+    width: 85%;
+  }
+`;
+
+interface Props {}
```

Comment:
Is this meant to be here?

Reply:
good call - I can remove that in the follow-up PR!